apps/pie-docs/src/__tests__/_11ty/shortcodes/__snapshots__/contentPageImage.test.js.snap

```diff
@@ -4,7 +4,7 @@ exports[`contentPageImage.js > creates the expected markup for the given config
 "<figure class=\\"c-someContainer-img c-contentImage c-contentImage--hasBackdrop\\">
         <div class=\\"c-contentImage-backdrop c-contentImage-default\\">
           <picture>
-            <source style=\\"--img-width: 200px;\\" media=\\"(max-width: 600px)\\" srcset=\\"some/path/to/mobile/image\\">
+            <source style=\\"--img-width--mobile: undefined;\\" media=\\"(max-width: 600px)\\" srcset=\\"some/path/to/mobile/image\\">
```

Comment:
I think if the variable populating this is undefined, we shouldn't add it to the style tag as it's just HTML bloat and not actually needed. Can you please update the code to prevent this?

Reply:
Great suggestion, thank you! I pushed the update 👍